src/components/SQFormDialog/SQFormDialog.js

```diff
@@ -88,7 +92,19 @@ SQFormDialog.propTypes = {
    * Yup validation schema shape
    * https://jaredpalmer.com/formik/docs/guides/validation#validationschema
    * */
-  validationSchema: PropTypes.object
+  validationSchema: PropTypes.object,
+  /** Whether or not the Cancel Confirm Dialog nedeeded */
+  showCancelConfirmDialog: PropTypes.bool,
+  /**
+   * Any Valid props for Cancel Confirmation Dialog
+   * title: Modal Title
+   * secondaryButtonText: The secondary button text (Button located on left side of Dialog)
+   * primaryButtonText: The primary button text (Button located on right side of Dialog)
+   * content: The content to be rendered in the dialog body
+   * https://jaredpalmer.com/formik/docs/guides/validation#validationschema
+   *
+   * */
+  cancelConfirmDialogProps: PropTypes.object
```

Comment:
Instead of a comment, the prop type itself should document the shape. Use prop type shape instead of prop type object please.